src/KeyConnector/KeyConnector.csproj

```diff
@@ -1,11 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
ℹ️ Only real change is here. The line breaks are to help with future merge conflicts.